src/main/java/com/cburch/logisim/file/XmlCircuitReader.java

```diff
@@ -56,10 +56,6 @@
 public class XmlCircuitReader extends CircuitTransaction {
 
   /**
-   * Get a circuit's component from a read XML file. If the component has a non-null "trackercomp"
```

Comment:
I would keep the first sentence and only remove the following ones, which actually refer to the separate tracking feature.

Reply:
The comment is useless, as it does not do what it says in there, there is no check on "trackercomponent" (being null or none-null that I could found anywhere.